Sources/Kitura/Router.swift

```diff
@@ -73,6 +73,22 @@ public class Router {
     /// and used to handle request's url parameters.
     fileprivate var parameterHandlers = [String : [RouterParameterHandler]]()
 
+    /// Contains the structures needed for swagger document generation
+    var swagger: SwaggerDocument
+
+    /// Computed property
+    ///
+    /// returns the current in-memory representation of the API as a json string.
+    public var swaggerJSON: String? {
+        get {
```

Comment:
Remove the `get { }`, it is only required if you have a `set { }` as well.

```swift
public var swaggerJSON: String? {
    do {
        return try self.swagger.serializeAPI(format: .json)
    } catch {
        return nil
    }
}
```

Reply:
done